src/groups/group.jl

```diff
@@ -562,12 +581,18 @@ inv_diff(G::AbstractDecoratorManifold, p)
 
 @trait_function inv_diff(G::AbstractDecoratorManifold, p, X)
 function inv_diff(::TraitList{<:IsGroupManifold}, G::AbstractDecoratorManifold, p, X)
-    Y = allocate_result(G, inv_diff, X, p)
-    return inv_diff!(G, Y, p, X)
+    return -adjoint_action(base_group(G), p, X)
 end
 
 @trait_function inv_diff!(G::AbstractDecoratorManifold, Y, p, X)
 
+# true only if tangent vectors are stored with the left-invariant convention
```

Comment:
Do we ensure that somehow or is this implicitly assumed to be the default?

Reply:
It is implicitly assumed to be the default.

Reply:
I think, then it would be better to document that somewhere in a doctoring and remove this comment.
The comment reads a bit like there is something left to do in the code for the case it is not stored like that.

Reply:
Done: put in the docstring.